test/integration.test.js

```
@@ -0,0 +1,95 @@
+  backgroundListeners = [];
+  contentListeners = [];
+  setupChrome();
+  jest.resetModules();
+});
+
+test("popup click triggers OpenAI call", async () => {
+  // Load background and content scripts
+  await import("../extension/background.js");
+  global.__loadingContent = true;
+  await import("../extension/content.js");
+  delete global.__loadingContent;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)